plugins/quickicon/eos/services/provider.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/**
+ * @package                 Joomla.Plugin
+ * @subpackage              Quickicon.Eos
+ * @copyright               (C) 2023 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license                 GNU General Public License version 2 or later; see LICENSE.txt
+ * @phpcs                   :disable PSR1.Classes.ClassDeclaration.MissingNamespace
+ */
```

Comment:
```suggestion
/**
 * @package     Joomla.Plugin
 * @subpackage  Quickicon.eos
 *
 * @copyright   (C) 2023 Open Source Matters, Inc. <https://www.joomla.org>
 * @license     GNU General Public License version 2 or later; see LICENSE.txt
 */
```
format
phpcs should not be necessary

Reply:
Removed @phpcs, and added back in blank line that was taken out when trying to debug a phpcs error!